bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -53,6 +53,20 @@
 else:
     import __builtin__ as builtins
 
+
+class _PoisonedDunderAll(object):
+    """Dummy class used to trigger an ImportError on wildcard imports if the
+    __all__attribute of a module is an instance of this class.
+    """
+
+    def __getitem__(self, _):
+        raise ImportError("Wildcard imports such as `from ROOT import *` are not possible in PyROOT")
```

Comment:
```suggestion
        raise ImportError(textwrap.dedent(
        """
        Wildcard import e.g. `from module import *` is bad practice, so it is disallowed in ROOT. Please import explicitly.
        """))
```

Reply:
Cool! I didn't know about this

Reply:
The best thing I can offer in favour of this statement is https://peps.python.org/pep-0020/#the-zen-of-python